### DISCUSSION TITLE
[Community Call] July 5, 2022

### DISCUSSION THREAD

**Discussion by natemoo-re** [withastro/roadmap#219] [Announcements]
## To Discuss

- **Announcement** Astro v1.0 still on track for end of month release
- **@natemoo-re** `@astrojs/mdx` [has been released](https://github.com/withastro/astro/tree/main/packages/integrations/mdx)
- **@tony-sull** `@astrojs/image` [has been released](https://github.com/withastro/astro/tree/main/packages/integrations/image)
- **@FredKSchott** https://github.com/withastro/rfcs/pull/215

_If there is something that you would like to see discussed, please comment below. We will prioritize items already on the agenda, but if there is time to spare we will open up the call to get attention on and discuss [other threads and feature requests](https://github.com/withastro/rfcs/discussions)._

**Comment by delucis** [comment]
# Community Call Minutes, 5 July 2022

Discord chat thread: https://discord.com/channels/830184174198718474/993938200458432653

### News 📰

@natemoo-re reports what’s new in the Astroverse:

- A quiet week with folks heads down bug bashing pre-v1, which is still on track for an end-of-July release!

- Help us ship Astro by reporting any issues you’re still experiencing 🚀

- Try out newer extensions like [`@astrojs/mdx`](https://github.com/withastro/astro/tree/main/packages/integrations/mdx) and [`@astrojs/image`](https://github.com/withastro/astro/tree/main/packages/integrations/image)! Let us know when you hit any bugs 🐛

- HTML streaming is now on by default in Astro projects — thanks @matthewp 🥳 

### RFCs/Issues to be discussed 💬

- 💅 #215

   @FredKSchott presents the ideas in the RFC
   
   - Astro already does things to `<style>` and `<script>` tags (hoisting, bundling etc.) so this is an extension of that (see [RFC0016](https://github.com/withastro/rfcs/blob/main/proposals/0016-style-script-defaults.md))

   - `<style src="...">` is not standard HTML — same syntax as Vue uses

   - A more friendly API than using ESM imports in Astro component frontmatter

   - Should also allow addressing a few bugs with importing CSS from npm packages that currently trip up ESM imports
     - @bholmesdev: These bugs will likely be addressed in Vite 3 in any case, so this is more of an aesthetic thing, but it’s a good aesthetic thing.

   - #### Dynamic `src`
     @aFuzzyBear: Can `src` be a dynamic string?
     @natemoo-re: Says `src` will need to be a static string like for scripts.

   - #### Scoping
     @natemoo-re: Would imports be scopes?
     @FredKSchott: Think `<script src="">` styles would be global like current ESM imports
     @matthewp: How would you opt in to scoping?
     @natemoo-re: We have a `is:scoped` directive that doesn’t do anything currently
     @FredKSchott & @natemoo-re: This might be out of scope for this PR

### Show and tell 🦔

- @natemoo-re demos a fun combination of Astro components and declarative Shadow DOM in custom elements (a.k.a. web components) — https://stackblitz.com/edit/github-4xpecr

- @bholmesdev demos a soon-to-be-shipped shared-state tabs component for the docs site 🎉
